src/api/controllers/PostController.ts

```diff
@@ -144,4 +144,12 @@ export class PostController {
   ): Promise<GetPostsResponse> {
     return { posts: await this.postService.getSuggestedPosts(user, limit) };
   }
+
+  @Get('searchSuggestions/:searchIndex/:postCount/')
+  async getSearchSuggestions(
+    @Params() params: { searchIndex: string; postCount: number }
+  ): Promise<{ postIds: string[] }> {
+    const postIds = await this.postService.getSearchSuggestions(params.searchIndex, Number(params.postCount));
+    return { postIds };
```

Comment:
usually the count / size the frontend wants is passed in through a query param (so like searchSuggestions/1?count=5 instead of searchSuggestions/1/5/)